### PR TITLE
Add style to Website

### PR DESCRIPTION
**Pull request by jheart-vic**
Hi, 
- Used the information detailed in Zeplin to style the website.
Kindly review

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+      </div>
+      <div className="line" />
+
+      <div className="content3">
+        <span className="currentChapter">CURRENT CHAPTER</span>
+        <span className="chapter">Chapter 17</span>
+        <button type="button" className="updateBtn">UPDATE PROGRESS</button>
+      </div>
+    </div>
```
Comment (youmari) [inline src/components/Book.js]: - [x] The book card has more height than the one in the Design please try to decrease the height and increase the width  of the card in order to match the design 
- Yours 
image
- Design 
image

File: src/components/Book.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        </div>
+
+      </div>
+      <div className="line" />
+
+      <div className="content3">
+        <span className="currentChapter">CURRENT CHAPTER</span>
+        <span className="chapter">Chapter 17</span>
+        <button type="button" className="updateBtn">UPDATE PROGRESS</button>
+      </div>
```
Comment (youmari) [inline src/components/Book.js]: - [x] please make sure to align  the current chapter and 17  and use the same color text for the button as in the Design in order to match it 
- Yours 
image
- Design 
image

File: src/components/Book.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+  return (
+
+    <div className="bookContents">
+      <div className="content1">
+        <div className="content1FirstChild">
+          <p className="genre">{book.category}</p>
+          <h2 className="title">{book.title}</h2>
+          <p className="author">{book.author}</p>
+        </div>
```
Comment (youmari) [inline src/components/Book.js:22]: - [x] please make sure to align the content as in the Design in order to match it
- Yours 
image
- Design 
image

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @jheart-vic , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ No linters error.
✔️ Details PR title and description.
✔️ Good commit messages.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Books.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+import Book from './Book';
+import AddBook from './AddBook';
+
+const Books = ({ books }) => (
+  <div>
+    <ul className="book-content">
+      {books.map((book) => <Book key={book.id} book={book} />)}
+    </ul>
+    <AddBook />
+  </div>
```
Comment (BenMukebo) [inline src/components/Books.js:12]: - [OPTIONAL] Kindly make sure your Styling is EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/) -Styling is EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/) :
- Correct the width and align properties 🤔  you could remove the left margin applied on each book container (`margin: 2.313rem 6.25rem 0;`) and the `text-align: center;` properly applied to the form and make sure the header, book, and form have the same width and the same align, Kindly correct this 🎨 🖌️ 👨🏻‍🎨 

- Your Design:

bookstore

- Original Design:

image

**Review by BenMukebo — APPROVED** [review]
### Hi @ , 👋 

# STATUS: Approved  🎉 

Great work on making the changes requested by a previous reviewer :1st_place_medal:  
 Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 




## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**